randovania/gui/preset_settings/standard_pickup_widget.py

```diff
@@ -95,7 +97,7 @@ def __init__(
             self.root_layout.addWidget(self.description_label, self.root_layout.rowCount(), 0, 1, -1)
 
         self.set_custom_fields_visible(False)
-        if self._pickup.must_be_starting:
+        if self._pickup.starting_condition == StartingEnum.MUST_BE_STARTING:
```

Comment:
This file also needs a change to not let you be able to set items as starting items that cant be any